public_html/wp-content/mu-plugins/blocks/assets/src/sessions/block-content.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+const { Disabled } = wp.components;
+const { Component } = wp.element;
+const { decodeEntities } = wp.htmlEntities;
+const { __ } = wp.i18n;
+
+/**
+ * Internal dependencies
+ */
+import { ItemTitle, ItemHTMLContent } from '../shared/block-content';
+import { tokenSplit, arrayTokenReplace, intersperse, listify } from '../shared/i18n';
+import GridContentLayout from '../shared/grid-layout/block-content';
+import FeaturedImage from '../shared/featured-image';
+
+function SessionSpeakers( { session } ) {
+	let speakerData = get( session, '_embedded.speakers', [] );
+
+	speakerData = speakerData.map( ( speaker ) => {
+		let { link = '', title = {} } = speaker;
+		title = title.rendered || __( 'Unnamed', 'wordcamporg' );
+
+		if ( ! link ) {
+			return decodeEntities( title.trim() );
+		}
+
+		return (
+			<a
+				key={ link }
+				href={ link }
+			>
+				{ decodeEntities( title.trim() ) }
+			</a>
+		);
+	} );
+
+	const speakers = arrayTokenReplace(
+		/* translators: %s is a list of names. */
+		tokenSplit( __( 'Presented by %s', 'wordcamporg' ) ),
+		[ listify( speakerData ) ]
+	);
+
+	return (
+		<div className="wordcamp-item-meta wordcamp-session-speakers">
+			{ speakers }
+		</div>
+	);
+}
+
+function SessionImage( { session } ) {
+	let image;
+
+	const url = get( session, '_embedded[\'wp:featuredmedia\'].media_details.sizes.thumbnail.source_url', '' );
+
+	if ( url ) {
+		image = (
+			<img
+				src={ url }
+				alt={ decodeEntities( session.title.rendered.trim() ) }
+				className={ classnames( 'wordcamp-session-image' ) }
+			/>
+		);
+	} else {
+		image = (
+			<div className="wordcamp-session-default-image" />
+		);
+	}
+
+	return image;
+}
+
+function SessionDetails( { session, show_meta, show_category } ) {
+	let meta, metaContent, category;
+	const terms = get( session, '_embedded[\'wp:term\']', [] ).flat();
+
+	if ( show_meta ) {
```

Comment:
`[SHOULD]` This condition feels like it's big enough that it should be modularized into its own function.

The same goes for `if ( show_category && session.session_category.length ) {` below.

Reply:
The corresponding checkbox for this is marked off in #44, but this is still marked as unresolved. If it's resolved, can you leave a comment w/ the commit that resolved it, and then mark as resolved?

Reply:
https://github.com/WordPress/wordcamp.org/pull/44/commits/15e377aa893c986810bd40a1b784cb975bb476c9